build.gradle

```diff
@@ -59,27 +117,27 @@ uploadArchives {
 // so you should probably use JCenter for dependency resolution in your own
 // project.
 repositories {
-  jcenter()
+    jcenter()
 }
 
 dependencies {
-  compile gradleApi()
-  compile localGroovy() //not needed for Java plugins
-  compile files('./umple-latest.jar')
+    compile gradleApi()
+    compile localGroovy() //not needed for Java plugins
+    compile files('./umple-latest.jar')
 }
 
 // The configuration example below shows the minimum required properties
```

Comment:
Remove this comment, or at least make it more accurate to the current code, not the template